src/Form/selectors.js

```diff
@@ -1,41 +1,40 @@
-    input[type=""][id*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
-    input[type=text][placeholder*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
-    input[type=""][placeholder*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
-    input:not([type])[placeholder*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
-    input[type=email]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
-    input[type=text][aria-label*=mail i],
-    input:not([type])[aria-label*=mail i],
-    input[type=text][placeholder*=mail i]:not([readonly]),
-    input[autocomplete=email]:not([readonly]):not([hidden]):not([disabled])
-`
+input:not([type])[name*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input[type=""][name*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input[type=text][name*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input:not([type])[id*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input:not([type])[placeholder*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input[type=""][id*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input[type=text][placeholder*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input[type=""][placeholder*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input:not([type])[placeholder*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input[type=email]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input[type=text][aria-label*=mail i],
+input:not([type])[aria-label*=mail i],
+input[type=text][placeholder*=mail i]:not([readonly]),
+input[autocomplete=email]:not([readonly]):not([hidden]):not([disabled])`
 
 // We've seen non-standard types like 'user'. This selector should get them, too
 const GENERIC_TEXT_FIELD = `
-input:not([type=button],
-[type=checkbox],
-[type=color],
-[type=date],
-[type=datetime-local],
-[type=datetime],
-[type=file],
-[type=hidden],
-[type=month],
-[type=number],
-[type=radio],
-[type=range],
-[type=reset],
-[type=search],
-[type=submit],
-[type=tel],
-[type=time],
-[type=url],
-[type=week])`
+input:not([type=button]),
+input:not([type=checkbox]),
+input:not([type=color]),
+input:not([type=date]),
+input:not([type=datetime-local]),
+input:not([type=datetime]),
+input:not([type=file]),
+input:not([type=hidden]),
+input:not([type=month]),
+input:not([type=number]),
+input:not([type=radio]),
+input:not([type=range]),
+input:not([type=reset]),
+input:not([type=search]),
+input:not([type=submit]),
+input:not([type=tel]),
+input:not([type=time]),
+input:not([type=url]),
+input:not([type=week])`
```

Comment:
The `:not(a, b, c)` was not working on Android, so had to revert to this more verbose selector. In the future, we will be able to simplify this stuff.